### ISSUE TITLE
Error when trying to install

### ISSUE THREAD

**Issue by Symbai** [MichalStrehovsky/sizoscope#7]
Running `dotnet tool install sizoscope --global --framework net7.0-windows` results in:

```
The settings file in the NuGet package of the tool is invalid: The settings file "DotnetToolSettings.xml" was not found in the package.
Error installing the "sizoscope" tool. Contact the author of the tool for support.
```

**Comment by MichalStrehovsky** [comment]
I assume you have a .NET 8 SDK? Something got broken in that SDK. The only workaround I know of right now is to install the package with a .NET 7 SDK and then upgrade to .NET 8...

I think it will be resolved once sizoscope is upgraded to .NET 8. I regret making this a .NET tool. Should have just put it on winget. Might do that for .NET 8...

**Comment by Symbai** [comment]
Yes I have a .NET 8 SDK. Installing the cross platform version worked fine.

**Comment by MichalStrehovsky** [comment]
This was fixed in #8 and shipped in 1.0.5.
